app/jobs/upvs/fetch_public_authority_edesks_list_job.rb

```diff
@@ -33,22 +33,18 @@ def self.source
 
   def each_row_as_attributes(csv_file, csv_options)
     CSV.foreach(csv_file, csv_options) do |row|
-      row = row.to_h.transform_keys { |k| k.to_s.gsub(/\p{Cf}/, '') }
+      row = row.to_h.transform_keys {|k| k.to_s.gsub(/\p{Cf}/, '')}
 
       yield(
-        uri: row.fetch('URI'),
-        cin: row['IČO'] || row['IČO'] || row.fetch('ICO'),
-        name: row['NAZOV INŠTITÚCIE'] || row.fetch('NÁZOV'),
-        street: row.fetch('ULICA'),
-        street_number: row.fetch('ČÍSLO'),
-        postal_code: row.fetch('PSČ'),
-        city: row.fetch('MESTO')
+          cin: row.fetch('ICO'),
+          uri: row.fetch('URI'),
+          name: row.fetch('Nazov')
       )
     end
   end
 
   def check_row_attributes(attributes)
-    uri, cin, name = attributes.slice(:uri, :cin, :name).values
+    cin, uri, name = attributes.slice(:cin, :uri, :name).values
```

Comment:
toto sa preco vymenilo?

Reply:
V strikture noveho datasetu sa zmenilo poradie, tak som to prehodil aj tu. Z dovodu konzistencie